File: fetch_artists.js
const https = require('https');

https.get('https://fanspherebackend.vercel.app/api/artists', (res) => {
  let data = '';
  res.on('data', (chunk) => {
    data += chunk;
  });
  res.on('end', () => {
    try {
      const json = JSON.parse(data);
      if (json.data && Array.isArray(json.data)) {
        const names = json.data.map(artist => artist.name);
        console.log(JSON.stringify(names));
      } else {
        console.log("No data found");
      }
    } catch (e) {
      console.error("Error parsing JSON:", e.message);
    }
  });
}).on('error', (err) => {
  console.error("Error fetching data:", err.message);
});

File: fetch_artists.py
import requests
import json

url = "https://fanspherebackend.vercel.app/api/artists"
try:
    response = requests.get(url)
    data = response.json()
    if "data" in data:
        names = [artist["name"] for artist in data["data"]]
        print(json.dumps(names))
    else:
        print("No data found")
except Exception as e:
    print(f"Error: {e}")
